packages/tocco-util/src/mockData/data/user_search_form.json

```diff
@@ -115,7 +115,22 @@
             "hidden": false,
             "scopes": [ ]
           },
-
+          {
```

Comment:
good that you tested it but i think you dont need to commit that field in the search form.